libfreerdp/core/gateway/rts.c

```diff
@@ -887,7 +887,7 @@ int rts_recv_flow_control_ack_pdu(rdpRpc* rpc, BYTE* buffer, UINT32 length)
 	BYTE ChannelCookie[16];
 
 	offset = 24;
-	offset += rts_flow_control_ack_command_read(rpc, &buffer[offset], length - offset,
```

Comment:
Even if the offset variable is not used beyond this point, updating it to point to the new offset after the last read operation makes it consistent with the rest. I don't think we want to remove such variable updates.